### PR TITLE
fix(pagewrapper.vue): tune background-size and background-position

### PR DESCRIPTION
**Pull request by furecool**
## Types of changes

* **Bugfix**

## Description
Adjust size and position of background image elements

## Steps to Test This Pull Request

1. Change `backgroundPosition`, and add `backgroundSize` properties in `tailwind.config`.
2. Add class `bg-bgsize` at `.page-wrapper` in `PageWrapper.vue.`

## Expected behavior
Adjust position of background image elements, and make them smaller.

## Related Issue
#250

### CODE REVIEW

File: tailwind.config.js [hunk 1]
```diff
@@ -13,14 +13,20 @@ module.exports = {
             ],
             serif: ['Noto Serif TC', '-apple-system', 'serif'],
         },
+        backgroundSize: {
+            auto: 'auto',
+            cover: 'cover',
+            contain: 'contain',
+            bgsize: '7%,5%,7%,7%,7%,7%,7%,5%,7%,7%,7%,7%',
```
Comment (flynnhou) [inline tailwind.config.js:20]: `bgsize` is a vague term. Everything under `backgroundSize` is considered a bgsize...
Can you elaborate more on this? Name things in a more self-explanatory manner.

**Review by flynnhou — CHANGES_REQUESTED** [review]

File: tailwind.config.js (hunk 1 above)
Comment (furecool) [reply tailwind.config.js]: It make sense. thanks.

File: tailwind.config.js [hunk 2]
```diff
@@ -13,14 +13,20 @@ module.exports = {
             ],
             serif: ['Noto Serif TC', '-apple-system', 'serif'],
         },
+        backgroundSize: {
+            auto: 'auto',
+            cover: 'cover',
+            contain: 'contain',
+            scale: '7%,5%,7%,7%,7%,7%,7%,5%,7%,7%,7%,7%',
```
[the remark this answers is not in the document]
Comment (furecool) [reply tailwind.config.js]: Got it. thanks.

**Review by flynnhou — APPROVED** [review]
LGTM!!

**Pull request merged**